reference/configuration/twig.rst

```diff
@@ -141,9 +141,9 @@ individually in the templates).
 
 If set to a string, the template contents are escaped using the strategy with
 that name. Allowed values are ``html``, ``js``, ``css``, ``url``, ``html_attr``
-and ``filename``. The default value is ``filename``. This strategy escapes
-contents according to the filename extension (e.g. it uses ``html`` for
-``*.html.twig`` templates and ``js`` for ``*.js.html`` templates).
+and ``name``. The default value is ``name``. This strategy escapes contents
+according to the filename extension (e.g. it uses ``html`` for ``*.html.twig``
```

Comment:
`according to the filename extension` -> `according to the template name extension` ?